py-polars/polars/series/series.py

```diff
@@ -1141,10 +1141,12 @@ def __array_ufunc__(
 
             args: list[int | float | np.ndarray[Any, Any]] = []
 
+            null_mask = self.is_null()
```

Comment:
Could you write this in terms of `is_not_null` and `&=`? Polars internally stores validity masks in this manner.